### ISSUE TITLE
PartDesign: Switching Draw-Styles causes several operations to not be applied

### ISSUE THREAD

**Issue by Samu152** [FreeCAD/FreeCAD#17318]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Problem description

If you

1. Create sketches in PartDesign
2. Switch the Draw-Style (for example to wireframe)
3. Perform an operation with the sketches (for example a loft) 
4. Switch the Draw-Style back to original

Then the contour of the resulting shape will be visible, but not applied to the model. See images below:

![AdditiveLoft](https://github.com/user-attachments/assets/d2722b95-8f4c-4971-9f49-34ceb5dc75e0)
![AdditivePipe](https://github.com/user-attachments/assets/32eec1a7-3d85-47bd-b478-6d36f4cd72d4)
![SubtractiveHelix](https://github.com/user-attachments/assets/de1edb66-3eb2-4b0c-a837-554905fd69e2)

Tested with:
- Additive Loft
- Subtractive Loft
- Additive Pipe
- Subtractive Helix (Additive Helix works as expected)
- Pad works as expected
- Revolution works as expected
- Hole works as expected
- Pocket works as expected

This only happens when switching to either: Points, Wireframe or Hidden Lines

### Full version info

```shell
OS: Windows 10 build 19045
Word size of FreeCAD: 64-bit
Version: 1.1.0dev.38977 (Git)
Build type: Release
Branch: main
Hash: facd8ca6e0303cf96297a7a7398185770c1a1e0b
Python 3.11.9, Qt 5.15.13, Coin 4.0.3, Vtk 9.2.6, OCC 7.7.2
Locale: German/Austria (de_AT)
Stylesheet/Theme/QtStyle: unset/FreeCAD Classic/Qt default
```

### Subproject(s) affected?

PartDesign

### Anything else?

gif OP made to display the bug in action: 

![17318](https://github.com/user-attachments/assets/a3c07eb2-4966-485a-a031-22540ea5f91d)

https://forum.freecad.org/viewtopic.php?t=91431

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by Samu152** [comment]
Just noticed a pattern. This seems to happen only with operations that don't apply the preview directly to the model. For example AdditiveLoft shows the preview as this yellow color you can see on the image. But a pad for example doesn't do that. It directly applies the changes to the model and is therefore unaffected

**Comment by maxwxyz** [comment]
@kadet1090 maybe that is something you've came across in https://github.com/FreeCAD/FreeCAD/pull/17249 ?

**Comment by luzpaz** [comment]
Bug confirmed in forum thread

**Comment by kadet1090** [comment]
> @kadet1090 maybe that is something you've came across in #17249 ?

No but I'll take a look within few days

**Comment by kadet1090** [comment]
Yes it is totally caused by how preview is done currently. #17249 does not have the same issue present.
